logging/logging.go

```diff
@@ -58,7 +58,7 @@ func DisableLogging() {
 
 // EnableStructuredLogs replaces all logging functions with structured logging
 // variants.
-func EnableStructuredLogs(logDebugStdout, verbose bool) (func(), error) {
+func EnableStructuredLogs(logDebugStdout, googleLogFormat, verbose bool) (func(), error) {
```

Comment:
The structured logging should have worked with Cloud Logging when it was introduced. So, I think of this as a bug fix and so it's fine to just update the format within this function without introducing another flag.

Reply:
Let's remove the flag, and alter the format inline, adding a comment explaining that it adheres to what Cloud Logging requires. How does that sound?

Reply:
great! I did it this way to ensure that I did not break any existing log metrics/alerts people may have created based on the original format. I did a force push of the update.